lib/pusher/push_notifications/user_id.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Pusher
+  module PushNotifications
+    class UserId
+      def max_user_id_length
```

Comment:
It's a bit odd to have these as functions as they will never change their value

Reply:
Fixed in 339db5d.